packages/web-react/src/components/Grid/README.md

```diff
@@ -17,14 +17,14 @@ Use Grid to build multiple column layouts. This Grid works on twelve column syst
 
 ## API
 
-| Name               | Type                                      | Default | Required | Description                         |
-| ------------------ | ----------------------------------------- | ------- | -------- | ----------------------------------- |
-| `cols`             | [`1` \| `2` \| `3` \| `4` \| `6` \| `12`] | —       | ✕        | Number of columns to use            |
-| `desktop`          | [`1` \| `2` \| `3` \| `4` \| `6` \| `12`] | —       | ✕        | Number of columns to use on desktop |
-| `elementType`      | HTML element                              | `div`   | ✕        | Element type to use for the Grid    |
-| `tablet`           | [`1` \| `2` \| `3` \| `4` \| `6` \| `12`] | —       | ✕        | Number of columns to use on tablet  |
-| `UNSAFE_className` | `string`                                  | —       | ✕        | Wrapper custom class name           |
-| `UNSAFE_style`     | `CSSProperties`                           | —       | ✕        | Wrapper custom style                |
+| Name               | Type                                                         | Default | Required | Description                                                                          |
+| ------------------ | ------------------------------------------------------------ | ------- | -------- | ------------------------------------------------------------------------------------ |
+| `cols`             | [`1` \| `2` \| `3` \| `4` \| `5` \| `6` \| `12` \| `object`] | —       | ✕        | Number of columns to use, use object to set responsive values                        |
```

Comment:
```suggestion
| `cols`             | [`1` \| `2` \| `3` \| `4` \| `5` \| `6` \| `12` \| `object`] | —       | ✕        | Number of columns to use, use object to set responsive values, e.g. `{ mobile: 1, tablet: 2, desktop: 3 }`                        |
```

I know that there is an example before this table, but in this case better to repeat it IMHO.